eng/Packages.Data.props

```diff
@@ -99,7 +99,7 @@
     <PackageReference Update="Azure.ResourceManager" Version="1.0.0-beta.7" />
 
     <!-- Other approved packages -->
-    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.5.8" />
+    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.5.9" />
```

Comment:
Let's wait for 2.5.10.

Reply:
Agreed.  That wasn't on the radar when I opened this.   😄